autoload/lsp/completion.vim

```diff
@@ -506,7 +505,7 @@ def g:LspOmniFunc(findstart: number, base: string): any
       count += 1
     endwhile
 
-    if lspserver.omniCompletePending
+    if lspserver.omniCompletePending || !lspserver->has_key('completeItems')
```

Comment:
The `lspserver.completeItems` field is set to an empty List in line 488 above, when the omni-completion function
is called with `findstart' set to true.  Under what conditions, the `completeItems` key will not be present
in `lspserver`?